pyfolio/plotting.py

```diff
@@ -582,6 +600,11 @@ def plot_rolling_returns(
                          'factor_returns.')
     elif volatility_match and factor_returns is not None:
         bmark_vol = factor_returns.loc[returns.index].std()
+        # TO-DO: @tweicki 'returns' probably needs to get updated to:
```

Comment:
I agree, if volatility_match is True, returns need to be changed/re-assigned as you proposed. That will fix the vol match plot.